editoast/src/views/timetable/stdcm.rs

```diff
@@ -329,142 +217,82 @@ async fn stdcm(
             .supported_signaling_systems
             .clone(),
         comfort: stdcm_request.comfort,
-        path_items,
+        path_items: stdcm_request.get_stdcm_path_items(conn, infra_id).await?,
         start_time: earliest_departure_time,
         trains_requirements: trains_requirements.clone(),
-        maximum_departure_delay,
-        maximum_run_time,
-        speed_limit_tag: stdcm_request.speed_limit_tags,
+        maximum_departure_delay: stdcm_request.get_maximum_departure_delay(simulation_run_time),
+        maximum_run_time: stdcm_request.get_maximum_run_time(simulation_run_time),
+        speed_limit_tag: stdcm_request.speed_limit_tags.clone(),
```

Comment:
Moving `train_requirements` and `speed_limit_tags` further down the field list may make the `clone()` unnecessary.